tests/sqllogictests/suites/mode/standalone/explain_native/project_set.test

```diff
@@ -26,7 +26,7 @@ AggregateFinal
     └── ProjectSet
         ├── output columns: [unnest (#2)]
         ├── estimated rows: 1.00
-        ├── set returning functions: unnest(CAST(fold_count.id (#0) AS Array(Int32 NULL) NULL))
+        ├── set returning functions: unnest(fold_count.id (#0))
```

Comment:
is this expected?

Reply:
yes, the data type is `Array(Int32 NULL) NULL`, so cast is not needed